exchanges/execute/src/execute.ts

```diff
@@ -0,0 +1,60 @@
+import { pipe, share, filter, map, fromPromise, mergeMap, merge } from 'wonka';
+import {
+  GraphQLSchema,
+  GraphQLFieldResolver,
+  GraphQLTypeResolver,
+  execute,
+} from 'graphql';
+import { Exchange, makeResult, makeErrorResult } from '@urql/core';
+
+interface ExecuteExchangeArgs {
+  schema: GraphQLSchema;
+  rootValue?: any;
+  contextValue?: any;
+  fieldResolver?: GraphQLFieldResolver<any, any>;
+  typeResolver?: GraphQLTypeResolver<any, any>;
+}
+/** Exchange for executing queries locally on a schema using graphql-js. */
+export const executeExchange = ({
+  schema,
+  rootValue,
+  contextValue,
+  fieldResolver,
+  typeResolver,
+}: ExecuteExchangeArgs): Exchange => ({ forward }) => {
+  return ops$ => {
+    const sharedOps$ = share(ops$);
+    const targetOperationTypes = ['query', 'mutation'];
+
+    const executedOps$ = pipe(
+      sharedOps$,
+      filter(f => targetOperationTypes.includes(f.operationName)),
```

Comment:
nit: It'd be great to just match the other exchanges in just comparing it directly, i.e. `operation => operation.operationName === 'query' || operation.operationName === 'subscription'`. Not that big of a deal though which one we go for.

Reply:
I've gone for:

```
const SUPPORTED_OPERATION_TYPES = ['query', 'mutation'];

......

.filter(operation =>
     SUPPORTED_OPERATION_TYPES.includes(operation.operationName)
),
```

Happy to change to your suggestion if you prefer.